File: Ex1.java
package test;
import java.util.Scanner;

class AreaCirculo{
    public static void main(String[] args){
        float pi = 3.14159;
        Scanner sc = new Scanner(System.in);
        float raio = sc.nextFloat();
        sc.nextLine();
        System.out.println("A=" + pi*(Math.pow(raio, 2)));
    }
}

File: Ex1.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main(void){
    double pi = 3.14159;
    double raio;
    scanf("%lf", &raio);
    getchar();
    double Area = pi * pow(raio, 2);
    printf("A=%.4lf", Area);
    return 0;
}
